### PR TITLE
ADCSAdministration PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1871153](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1871153)

One new cmdlet. Other changes are Input and Output.

### CODE REVIEW

File: docset/winserver2022-ps/adcsadministration/ADCSAdministration.md [hunk 1]
```diff
@@ -26,7 +26,10 @@ Adds a certificate template to the CA.
 ### [Backup-CARoleService](./Backup-CARoleService.md)
 Backs up the CA database and private key information.
 
-### [Confirm-CAEndorsementKeyInfo](./Confirm-CAEndorsementKeyInfo.md)
+### [Confirm-CAAttestationIdentityKeyInfo](Confirm-CAAttestationIdentityKeyInfo.md)
+Checks whether the local certification authority trusts secure hardware for identity attestation.
```
Comment (thirunadh) [inline docset/winserver2022-ps/adcsadministration/ADCSAdministration.md:30]: can you use CA here to be inline with all other descriptions.

File: docset/winserver2022-ps/adcsadministration/Confirm-CAAttestationIdentityKeyInfo.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+### Certificate
+```
+Confirm-CAAttestationIdentityKeyInfo [-Certificate] <X509Certificate2> [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Confirm-CAAttestationIdentityKeyInfo** cmdlet checks whether the local certification authority (CA) trusts secure hardware, such as a Trusted Platform Module (TPM), for identity attestation. The Attestation Identity Key replaces the Endorsement Key as an identity for the TPM. An Attestation Identity Key is permanently embedded in the security hardware. The public portion of the key helps to recognize genuine security hardware.
+
+This cmdlet verifies whether the public certificate connects through a certificate chain to an anchor that the CA trusts. Specify an X509 certificate by using the **Certificate** parameter.
```
Comment (thirunadh) [inline docset/winserver2022-ps/adcsadministration/Confirm-CAAttestationIdentityKeyInfo.md:30]: Please update it to - "This cmdlet verifies whether the Attestation Identity Key (AIK) public certificate ..."

File: docset/winserver2022-ps/adcsadministration/Confirm-CAAttestationIdentityKeyInfo.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+This cmdlet checks whether the attestation identity public key exists as a file in a folder configured at the local CA for key attestation. Specify the public key by using the **PublicKeyHash** parameter.
```
Comment (thirunadh) [inline docset/winserver2022-ps/adcsadministration/Confirm-CAAttestationIdentityKeyInfo.md:32]: use capital letters for attestation identity - "... Attestation Identity Key (AIK) public key exists ..."

**Review by thirunadh — CHANGES_REQUESTED** [review]

**Review by thirunadh — APPROVED** [review]
Looks good to me. Thanks.

**Comment by TimShererWithAquent** [comment]
Hi @tiburd - I got SME approval for this one. Ready for review.

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by TimShererWithAquent** [comment]
@tiburd It looks like we have three approvals and are ready for merge.

**Pull request merged**